packages/js/product-editor/src/blocks/generic/text-area/toolbar/toolbar-button-alignment/index.tsx

```diff
@@ -38,7 +38,7 @@ export const ALIGNMENT_CONTROLS = [
 	},
 ];
 
-export default function AligmentToolbarButton( {
+export default function AlignmentToolbarButton( {
```

Comment:
Similar to my last comment, because this function is being exported, other's could be importing and using this (like any WooCommerce extensions). So while we've addressed that within Woo Core itself in this PR, not sure what the best practice is in renaming these public functions, if we need to do some sort of deprecation or not

Reply:
That said, code changes look good here so I think we can proceed to the next steps here and see if the Woo team has advice on the best way to handle these sorts of function naming changes

Reply:
This function is not exported as a named module (the import name does not matter), so I don't think we need any deprecation handling. Having said that, I'm going to check with @gigitux to be 100% sure here. 🙏

Reply:
> This function is not exported as a named module (the import name does not matter), so I don't think we need any deprecation handling. Having said that, I'm going to check with @gigitux to be 100% sure here. 🙏

💯 agree with you! It is not an issue! 💪